lib/Runtime/Language/ProfilingHelpers.cpp

```diff
@@ -994,6 +994,10 @@ using namespace Js;
                         : PatchGetMethod ;
                 value = (!Root && !Method) ? PatchGetValue(functionBody, inlineCache, inlineCacheIndex, object, propertyId, thisObject) :
                     PatchGet(functionBody, inlineCache, inlineCacheIndex, object, propertyId);
+                if (VarIs<JavascriptBigInt>(value))
```

Comment:
I'm surprised that this type would need special handling here. If you override the instance method CloneToScriptContext inherited from RecyclableObject, does it maybe just work?

Reply:
roll it back and defer to a future PR